src/main/java/lulu/Lulu.java

```diff
@@ -0,0 +1,73 @@
+package lulu;
+
+import lulu.command.Parser;
+import lulu.command.Command;
+import lulu.command.LoadCommand;
+
+import lulu.exception.LuluException;
+
+public class Lulu {
+    private static final String NAME = "./data/lulu.txt";
+    private TaskList tasks;
+    private UI ui;
```

Comment:
Maybe perhaps consider changing your class name to  `Ui` instead? Since Ui is an abbreviation.
```suggestion
    private Ui ui;
```